### DISCUSSION TITLE
Feature request PHANTOM

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#39] [PHANTOM] [answered]
Anonymous

In addition to the 12 phantoms available within the folder, is it also possible to access to the rest of the database containing 370 phantoms?

**Comment by choonsiklee** [comment, marked as answer]
The full body size phantom library (n=362) is now available from Onedrive for users with software transfer agreement completed.

**Comment by ncidoseteam** [comment]
include arms-included body size dependent phantom library to the repository
